src/main/scala/jsonparser/JsonParser.scala

```diff
@@ -0,0 +1,65 @@
+package jsonparser
+
+import net.liftweb.json.{DefaultFormats, parse}
+
+import java.sql.{Connection, DriverManager}
+import java.util.Properties
+import scala.collection.mutable.ListBuffer
+import scala.io.Source
+
+case class Language(name: String, createdBy: String, firstAppeared: Int, description: String)
+
+object JsonParser extends App {
+  val DbConfFilename = "db.properties"
+  val JsonFilename = "programming_languages.json"
+  implicit val formats = DefaultFormats
+
+  val db = new Properties()
+  db.load(Source.fromResource(DbConfFilename).bufferedReader())
+  var conn: Connection = _
+  try {
```

Comment:
```suggestion
  try {
    Class.forName("com.mysql.cj.jdbc.Driver")
```
Doesn't work for me without this line

Reply:
it's weird as Class.forName is not required since JDBC 4.0

Reply:
anyway, added Class.forName